va/va.h

```diff
@@ -1477,10 +1509,14 @@ typedef struct _VAEncMiscParameterRateControl
              * The temporal layer that the rate control parameters are specified for.
              */
             unsigned int temporal_id : 8;
-            unsigned int reserved : 17;
+            unsigned int cfs_I_frames : 1; /* I frame also follows CFS */
+            unsigned int enable_parallel_brc    : 1;
+            unsigned int enable_dynamic_scaling : 1;
```

Comment:
will change the code and add an attribute with comments
    /**
     * \brief Dynamic Scaling Attribute. Read-only.
     * This attribute conveys whether encoder is capable to determine dynamic frame
     * resolutions adaptive to bandwidth utilization and processing power, etc., as well
     * as capable to scale raw source and reference frames.
     * It is a boolean value 0 - unsupported, 1 - supported.
     * If it is supported, app may enable it by setting enable_dynamic_scaling in 
     * VAEncMiscParameterRateControl.
     */
     VAConfigAttribEncDynamicScaling        = 30,